masterlist.yaml

```diff
@@ -26498,6 +26498,9 @@ plugins:
       - <<: *deletePlugin
         condition: 'active("RUSTIC SOULGEMS - (Uns|S)orted\.esp") and active("Requiem - Rustic Soulgems( - ISC)?.esp")'
         subs: [ 'the Requiem patch for Rustic Soulgems' ]
+    tag: [ Graphics ]
+  - name: 'RUSTIC SOULGEMS - Sorted.esp'
+    url: [ 'https://www.nexusmods.com/skyrim/mods/63766' ]
```

Comment:
@Kaenguru-Knobel-Kette This location is redundant as the plugin inherits it from the above regex entry. Looks good otherwise.

```suggestion
```